raiden/network/proxies/payment_channel.py

```diff
@@ -216,3 +228,20 @@ def all_events_filter(
             to_block=to_block,
         )
         return channel_filter, unlock_filter
+
+    def check_outdated_channel(self):
```

Comment:
`check_outdated_channel()` -> `check_for_outdated_channel()`. Also add a docstring here explaining what this function does and why it's required.